main.py

```diff
@@ -0,0 +1,53 @@
+import math
+import csv
+import requests
+from bs4 import BeautifulSoup
+import sys
+
+
+def get_distance(loc1, loc2):
+    return round(math.sqrt((loc1[0] - loc2[0]) * (loc1[0] - loc2[0]) + (loc1[1] - loc2[1]) * (loc1[1] - loc2[1])), 4)
+
+
+def bye(message):
+    print(message)
+    quit()
+
+
+#  extra idea for the future - use this to get your location live
+
+# send_url = "http://api.ipstack.com/check?access_key=da96285c376c412b6bab56d3dfaa4961"
+# geo_req = requests.get(send_url)
+# geo_json = json.loads(geo_req.text)
+# my_location = [float(geo_json['latitude']), float(geo_json['longitude'])]
```

Comment:
Would this apply for the points being on a plane?

Reply:
Would you use your plane coordinates to find a nearby coffee shop?

Reply:
For this certain case where we consider that all coordinates rely on the plane is viable. If we are to make this to be a real life problem solver we should take in consider a lot of more factors such as:
- pathfinding in the city for getting from your location to the coffee shops
- daily trafic changes and cases where a street gets closed for construction
etc..